packages/contracts-bedrock/contracts/libraries/Clone.sol

```diff
@@ -0,0 +1,120 @@
+// SPDX-License-Identifier: BSD
+pragma solidity ^0.8.15;
+
+/**
+ * @title Clone
+ * @author zefram.eth, Saw-mon & Natalie, clabby
+ * @notice Provides helper functions for reading immutable args from calldata
+ * @dev MODIFICATIONS:
```

Comment:
this modifications is only useful relative to something so the original link should be included as well